onnxscript/optimizer/_constant_folding.py

```diff
@@ -119,7 +119,11 @@ def evaluate(self, domain: str, op: str, version: int, *args, **kwargs) -> Any:
         evaluator = self.get_evaluator(domain, op, version)
         if evaluator is None:
             return None
-        return evaluator(*args, **kwargs)
+        try:
+            return evaluator(*args, **kwargs)
+        except Exception as e:
+            logger.debug("Evaluation failed: %s", e)
```

Comment:
```suggestion
            logger.warning("Evaluation failed: %s", e)
```

maybe an warning is appropriate